app/views/acceptItemInformationList.scala.html

```diff
@@ -44,6 +39,5 @@
             <li>@messages("acceptItemInformationList.askfor.item2")</li>
         </ul>
 
-        @components.submit_button()
+        @components.button_link(messages("site.continue"), routes.InformationAboutYourItemController.onPageLoad(NormalMode).url)
```

Comment:
I'd make use of the `Navigator` if you can, it enables us to change the page order etc without too much effort

Reply:
Can't find an example of a page doing this directly.  Can the Navigator be injected into the page?  And what would it do to turn a `UserAnswers => Call` into a url string?

Could go back to calling another GET endpoint on the Controller and having that re-direct (like the POST used to do).   Is that worth it?

Reply:
`Call.url` converts a Call to a URL String

Up to you, if its awkward to do there's no need. We just lose a bit of flexibility if we don't use it